### PR TITLE
Kusto 2020-06-14 API version

### PR DESCRIPTION
**Pull request by astauben**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by astauben** [comment]
@leni-msft identical PR to the one I opened to the private repo

**Comment by orhasban** [comment]
@anthony-c-martin could you please either review the PR or assign some else?

File: specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2020-06-14/kusto.json [hunk 1]
```diff
@@ -0,0 +1,4329 @@
...
+          "200": {
+            "description": "OK -- Operation to check the kusto resource name availability was successful.",
+            "schema": {
+              "$ref": "#/definitions/DataConnectionValidationListResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Kusto/clusters/{clusterName}/databases/{databaseName}/dataConnectionValidationAsync": {
```
Comment (pilor) [inline specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2020-06-14/kusto.json:1949]: you shouldn't introduce a new API here.  Just start supporting an async response in this api-version and future api-versions for the existing API.

**Review by pilor — CHANGES_REQUESTED** [review]

File: specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2020-06-14/kusto.json (hunk 1 above)
Comment (astauben) [reply specification/azure-kusto/resource-manager/Microsoft.Kusto/stable/2020-06-14/kusto.json]: @pilor  Thanks for the quick review! I've updated the relevant files.

**Review by pilor — APPROVED** [review]

**Comment by akning-ms** [comment]
Hi @astauben  Ping me if you need to merge this PR as you put Do not Merge label

**Comment by astauben** [comment]
> Hi @astauben Ping me if you need to merge this PR as you put Do not Merge label

Hi @akning-ms , please merge the PR. Thanks!

**Comment by astauben** [comment]
Thanks!

**Pull request merged**